logical_router_static_route.go

```diff
@@ -36,14 +37,19 @@ func (odbi *ovndb) lrsrAddImp(lr string, ip_prefix string, nexthop string, outpu
 	if err != nil {
 		return nil, err
 	}
+
 	row := make(OVNRow)
 	row["ip_prefix"] = ip_prefix
 	row["nexthop"] = nexthop
-	if len(output_port) > 0 {
+	if len(output_port) == 1 {
 		row["output_port"] = output_port
+	} else if len(output_port) > 1 {
+		return nil, fmt.Errorf("only 1 output port is allowed")
 	}
-	if len(policy) > 0 {
-		row["policies"] = policy
+	if len(policy) == 1 {
+		row["policy"] = policy
+	} else {
```

Comment:
else if len() > 1

Reply:
Maybe it is better to use the same convention as LRSRDelEx(), i.e. make the parameter *string instead of []string